packages/release/usr-share-licenses.mount.in

```diff
@@ -2,6 +2,7 @@
 Description=License files
 Conflicts=umount.target
 Before=local-fs.target umount.target
+ConditionPathExists=!/proc/vmcore
```

Comment:
I'm OK with this, but for some of these mounts I wonder if we should just move them to happen in `preconfigured.target` where they can run in parallel with other system configuration. Then they wouldn't be automatically pulled in by `local-fs.target`, and we wouldn't need this condition.